codes/csharp/chapter_array_and_linkedlist/Array.cs

```diff
@@ -1,14 +1,14 @@
-/*
-* File: Array.cs
-* Created Time: 2022-12-14
-* Author: mingXta (1195669834@qq.com)
-*/
- 
+// File: Array.cs
+// Created Time: 2022-12-14
+// Author: mingXta (1195669834@qq.com)
+
 namespace hello_algo.chapter_array_and_linkedlist
 {
     public class Array
     {
-        /* 随机返回一个数组元素 */
+        /// <summary>
```

Comment:
Thank you for updating the function headers to the C# standard format. However, it is recommended to use the `/* ... */` format to make the codes more compact because the codes are also shown in markdown files.

Reply:
`///` can will give better ide support and I will use `/* ... */` in the md file,Is this okay？

Reply:
Yes, it is up to you~ However, in my opinion, it will make the work harder cause should modify comments for every function.